src/schemas/json/package.json

```diff
@@ -828,6 +831,179 @@
+              "items": {
+                "type": "string"
+              }
+            },
+            "allowedVersions": {
+              "description": "Unmet peer dependency warnings will not be printed for peer dependencies of the specified range.",
+              "type": "object"
+            },
+            "allowAny": {
+              "description": "Any peer dependency matching the pattern will be resolved from any version, regardless of the range specified in \"peerDependencies\".",
+              "type": "array",
+              "items": {
+                "type": "string"
+              }
+            }
+          },
+          "additionalProperties": false
+        },
+        "neverBuiltDependencies": {
+          "description": "A list of dependencies to skip the builds.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "onlyBuiltDependencies": {
+          "description": "A list of dependencies to skip the builds.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "onlyBuiltDependenciesFile": {
+          "description": "Specifies a JSON file that lists the only packages permitted to run installation scripts during the pnpm install process.",
+          "type": "string"
+        },
+        "allowedDeprecatedVersions": {
+          "description": "A list of deprecated versions that the warnings are suppressed.",
+          "type": "object"
+        },
+        "patchedDependencies": {
+          "description": "A list of dependencies that are patched.",
+          "type": "object"
+        },
+        "allowNonAppliedPatches": {
+          "description": "When true, installation won't fail if some of the patches from the \"patchedDependencies\" field were not applied.",
+          "type": "boolean"
+        },
+        "updateConfig": {
+          "type": "object",
+          "properties": {
+            "ignoreDependencies": {
+              "description": "A list of packages that should be ignored when running \"pnpm outdated\" or \"pnpm update --latest\".",
+              "type": "array",
+              "items": {
+                "type": "string"
+              }
+            }
+          },
+          "additionalProperties": false
+        },
+        "auditConfig": {
+          "type": "object",
+          "properties": {
+            "ignoreCves": {
+              "description": "A list of CVE IDs that will be ignored by \"pnpm audit\".",
+              "type": "array",
+              "items": {
+                "type": "string",
+                "pattern": "^CVE-\\d{4}-\\d{4,7}$"
+              }
+            },
+            "ignoreGhsas": {
+              "description": "A list of GHSA Codes that will be ignored by \"pnpm audit\".",
+              "type": "array",
+              "items": {
+                "type": "string",
+                "pattern": "^GHSA(-[23456789cfghjmpqrvwx]{4}){3}$"
```

Comment:
The format is described here: https://docs.github.com/en/code-security/security-advisories/working-with-global-security-advisories-from-the-github-advisory-database/about-the-github-advisory-database#about-ghsa-ids